libraries/core-react/src/components/Pagination/Pagination.tsx

```diff
@@ -82,7 +82,7 @@ export type PaginationProps = {
   onChange?: (event: MouseEvent | KeyboardEvent, page: number) => void
```

Comment:
Suggest using the Reacts type for `event` in our custom event handler.

Reply:
This caused some problems so we agreed to let this fix be good for now. We should look at these types to conform with the native event types in a few more of our components.